cmd/lncli/cmd_invoice.go

```diff
@@ -60,6 +60,14 @@ var addInvoiceCommand = cli.Command{
 				"specified, an expiry of " +
 				"86400 seconds (24 hours) is implied.",
 		},
+		cli.Uint64Flag{
+			Name: "cltv_expiry_delta",
+			Usage: "The minimum CLTV delta to use for the final " +
+				"hop. If this is set to 0, the default value " +
+				"is used. The default value for " +
+				"cltv_expiry_delta is configured by the " +
+				"bitcoin.timelockdelta' option.",
```

Comment:
nit: missing leading single quote of config option name, same below.

Reply:
Thanks for catching this! Pushed a fixup commit: https://github.com/lightningnetwork/lnd/pull/8491/commits/e14c25ae966dcbd1c7b0c52bacb5d309fd3004a0